sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -57,6 +58,51 @@
 from sklearn.linear_model._coordinate_descent import _set_order
 from sklearn.utils import check_array
 
+# FIXME: 'normalize' to be removed in 1.2
+filterwarnings_normalize = pytest.mark.filterwarnings(
+    "ignore:'normalize' was deprecated in version 1.0"
+)
+
+
+# FIXME: 'normalize' to be removed in 1.2
+@pytest.mark.parametrize(
+    "CoordinateDescentModel",
+    [
+        ElasticNet,
+        Lasso,
+        LassoCV,
+        ElasticNetCV,
+        MultiTaskElasticNet,
+        MultiTaskLasso,
+        MultiTaskElasticNetCV,
+        MultiTaskLassoCV,
+    ],
+)
+@pytest.mark.parametrize(
+    "normalize, n_warnings", [(True, 1), (False, 1), ("deprecated", 0)]
+)
+def test_assure_warning_when_normalize(CoordinateDescentModel, normalize, n_warnings):
+    # check that we issue a FutureWarning when normalize was set
+    rng = check_random_state(0)
+    n_samples = 200
+    n_features = 2
+    X = rng.randn(n_samples, n_features)
+    X[X < 0.1] = 0.0
+    y = rng.rand(n_samples)
+
+    if "MultiTask" in CoordinateDescentModel.__name__:
+        y = np.stack((y, y), axis=1)
+
+    if "CV" in CoordinateDescentModel.__name__ and normalize is True:
+        n_warnings += 1
```

Comment:
An inline comment to explain what additional warning is expected here would be welcome.